test/Microsoft.AspNetCore.DataProtection.Test/DockerUtilsTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNetCore.DataProtection.Internal;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace Microsoft.AspNetCore.DataProtection.Test
+{
+    public class DockerUtilsTests
```

Comment:
These lines are the actual contents of a /proc/self/mounts file with a volume mounted folder inside a microsoft/dotnet docker image.